### PR TITLE
k8s-stack: check `notifier.blackhole` flag if no notifiers are set in vmalert

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
https://victoriametrics.slack.com/archives/CTC488T1P/p1726776490053099?thread_ts=1725540517.623349&cid=CTC488T1P

### CODE REVIEW

File: charts/victoria-metrics-k8s-stack/templates/_helpers.tpl [hunk 1]
```diff
@@ -256,9 +256,6 @@ If release name contains chart name it will be used as a full name.
     {{- $_ := set $vmAlertRemotes "notifiers" . }}
   {{- end }}
   {{- $spec := deepCopy (omit $Values.vmalert.spec "notifiers") | mergeOverwrite $vmAlertRemotes | mergeOverwrite $vmAlertTemplates | mergeOverwrite $spec }}
-  {{- if not (or (hasKey $spec "notifier") (hasKey $spec "notifiers") (hasKey $spec "notifierConfigRef")) }}
-    {{- fail "`notifier`, `notifiers` or `notifierConfigRef` should be set for vmalert"}}
-  {{- end }}
```
Comment (zekker6) [inline charts/victoria-metrics-k8s-stack/templates/_helpers.tpl:261]: Wdyt about enforcing user to set `notifier.blackhole` if there are no other notifiers set?
This allows to make sure that user explicitly wants to blackhole all notifications and did not forget to set notifier by acccident.

Comment (AndrewChubatiuk) [reply charts/victoria-metrics-k8s-stack/templates/_helpers.tpl]: updated PR

**Review by zekker6 — DISMISSED** [review]
LGTM

See a suggestion for a changelog entry, it should make it easier to understand for the end users.

File: charts/victoria-metrics-k8s-stack/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 - Added VMAuth to k8s stack. See [this issue](https://github.com/VictoriaMetrics/helm-charts/issues/829)
 - Fixed ETCD dashboard
 - Use path prefix from args as a default path prefix for ingress. Related [issue](https://github.com/VictoriaMetrics/helm-charts/issues/1260)
+- Force setting blackhole notifier if no notifiers are set
```
Comment (zekker6) [inline charts/victoria-metrics-k8s-stack/CHANGELOG.md:6]:
```suggestion
- Allow using vmalert without notifiers configuration. Note that it is required to use `.vmalert.spec.extraArgs["notifiers.blackhole"]: true` in order to start vmalert with a blackhole configuration.
```

**Comment by AndrewChubatiuk** [comment]
updated error in helm and changelog, please approve again

**Review by Haleygo — APPROVED** [review]

**Pull request merged**